test/heartbeat/listener.js

```diff
@@ -510,7 +548,10 @@ describe('test/heartbeat/listener.js >', () => {
     it('when context is updated from encryption = false to encryption = ' +
       'true, decryption should happen', (done) => {
       const res = {
-        heartbeatInterval: 50,
+        collectorConfig: {
+          heartbeatInterval: 50,
+          maxSamplesPerBulkRequest: 10,
+        },
         generatorsAdded: [
           {
             name: 'Core_Trust4_With_Encryption',
```

Comment:
Looks like you missed tests for the buffered queue related changes in the `heartbeatResponse`.

Reply:
I did not get this. Which test ?